eos-tech-support/eos-install-firefox

```diff
@@ -0,0 +1,341 @@
+Name[es]=Abrir una ventana nueva
+Name[fi]=Avaa uusi ikkuna
+Name[fr]=Ouvrir une nouvelle fenêtre
+Name[gl]=Abrir unha nova xanela
+Name[he]=פתיחת חלון חדש
+Name[hr]=Otvori novi prozor
+Name[hu]=Új ablak nyitása
+Name[it]=Apri una nuova finestra
+Name[ja]=新しいウィンドウを開く
+Name[ko]=새 창 열기
+Name[ku]=Paceyeke nû veke
+Name[lt]=Atverti naują langą
+Name[nb]=Åpne et nytt vindu
+Name[nl]=Nieuw venster openen
+Name[pt]=Abrir nova janela
+Name[pt_BR]=Abrir nova janela
+Name[ro]=Deschide o fereastră nouă
+Name[ru]=Новое окно
+Name[sk]=Otvoriť nové okno
+Name[sl]=Odpri novo okno
+Name[sv]=Öppna ett nytt fönster
+Name[tr]=Yeni pencere aç 
+Name[ug]=يېڭى كۆزنەك ئېچىش
+Name[uk]=Відкрити нове вікно
+Name[vi]=Mở cửa sổ mới
+Name[zh_CN]=新建窗口
+Name[zh_TW]=開啟新視窗
+Exec=${DEST_DIR}/firefox/firefox -new-window
+OnlyShowIn=Unity;
+
+[Desktop Action NewPrivateWindow]
+Name=Open a New Private Window
+Name[ar]=افتح نافذة جديدة للتصفح الخاص
+Name[ca]=Obre una finestra nova en mode d'incògnit
+Name[de]=Ein neues privates Fenster öffnen
+Name[es]=Abrir una ventana privada nueva
+Name[fi]=Avaa uusi yksityinen ikkuna
+Name[fr]=Ouvrir une nouvelle fenêtre de navigation privée
+Name[he]=פתיחת חלון גלישה פרטית חדש
+Name[hu]=Új privát ablak nyitása
+Name[it]=Apri una nuova finestra anonima
+Name[nb]=Åpne et nytt privat vindu
+Name[ru]=Новое приватное окно
+Name[sl]=Odpri novo okno zasebnega brskanja
+Name[tr]=Yeni bir pencere aç
+Name[uk]=Відкрити нове вікно у потайливому режимі
+Name[zh_TW]=開啟新隱私瀏覽視窗
+Exec=${DEST_DIR}/firefox/firefox -private-window
+OnlyShowIn=Unity;
+EOF
+
+eos-add-to-desktop firefox
+
+# Download and install Java
+download_via_browser "Java" ${JAVA_PATTERN} ${JAVA_URL}
+filename=`ls ${JAVA_PATTERN}`
+
+echo "Installing ${filename}"
+tar -xf ${filename} -C ${DEST_DIR}
+
+rm ${filename}
+
+mkdir -p ~/.mozilla/plugins
+rm -f ~/.mozilla/plugins/libnpjp2.so
+ln -s ${DEST_DIR}/jre*/lib/amd64/libnpjp2.so ~/.mozilla/plugins
+
+# Download and install Flash
+echo "Downloading ${FLASH_FILENAME}"
+${WGET_COMMAND} "${FLASH_URL}" -O "${TEMP_DIR}/${FLASH_FILENAME}" || exit_with_error "Failed to download ${FLASH_FILENAME}"
+
+echo "Verifying ${TEMP_DIR}/${FLASH_FILENAME}"
+echo "${FLASH_SHA256SUM} ${TEMP_DIR}/${FLASH_FILENAME}" | sha256sum -c > /dev/null 2>&1 \
+   || exit_with_error "sha256sum mismatch ${TEMP_DIR}/${FLASH_FILENAME}"
+echo "Installing ${TEMP_DIR}/${FLASH_FILENAME}"
+cd ${TEMP_DIR}
+unzip ${FLASH_FILENAME} || exit_with_error "Cannot unpack plugin"
+flash_inner_path=$(echo -n ${FLASH_VERSION} | sed -r "s/([0-9]+)\.([0-9]+)\.([0-9]+)\.([0-9])/\1_\2_r\3_\4/")
+flash_inner_file=$(echo -n ${FLASH_VERSION} | sed -r "s/([0-9]+)\.([0-9]+)\.([0-9]+)\.([0-9])/\1_\2r\3_\4/")
+tar zxf ${flash_inner_path}_64bit/flashplayer${flash_inner_file}_linux.x86_64.tar.gz || exit_with_error "Cannot unpack plugin"
+flash_dir=${DEST_DIR}/flash
+mkdir -p ${flash_dir}
+install -m 644 libflashplayer.so ${flash_dir}
+echo -n "${FLASH_VERSION}" > ${flash_dir}/VERSION.txt
```

Comment:
I copied it from the old flash plugin downloader, and thought it might be helpful, but I suppose if someone really wants to check which version they have they can do so via the browser.  I'll remove it.